### PR TITLE
[PyTorch] Add RNN Scratch

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1088/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1088/

File: chapter_recurrent-neural-networks/rnn-scratch.md [hunk 1]
```diff
@@ -47,24 +71,51 @@ def get_params(vocab_size, num_hiddens, ctx):
...
     return params
 ```
 
+```{.python .input}
+#@tab pytorch
+def get_params(vocab_size, num_hiddens, device):
+    num_inputs = num_outputs = vocab_size
+
+    def normal(shape):
+        return torch.randn(size=shape, device=device)*0.01
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/rnn-scratch.md:91]: adds space around operator

Comment (astonzhang) [reply chapter_recurrent-neural-networks/rnn-scratch.md:91]: otherwise LGTM

Comment (AnirudhDagar) [reply chapter_recurrent-neural-networks/rnn-scratch.md:91]: sorry missed that.

**Pull request merged**